vignettes/creating_linters.Rmd

```diff
@@ -157,29 +226,34 @@ such as the type, line number, column number, e.g.
 ```r
 expect_lint("blah=1",
   list(message = "assignment", line_number = 1, column_number = 5, type = "style"),
-  assignment_linter())
+  assignment_linter()
+)
 ```
 
 Finally, it is a good idea to test that your linter reports multiple lints if
 needed, e.g.
+
 ```r
 expect_lint("blah=1; blah=2",
   list(
     list(line_number = 1, column_number = 5),
     list(line_number = 1, column_number = 13),
   )
-  assignment_linter())
+  assignment_linter()
+)
 ```
 
 It is always better to write too many tests rather than too few.
 
 ## Adding your linter to the default_linters ##
-If your linter is non-project specific you can add it to `default_linters`.
-This object is created in the file `zzz.R`.  The name ensures that it will always run after all
-the linters are defined.  Simply add your linter name to the `default_linters`
+
+If your linter is not project-specific you can add it to `default_linters`.
+This object is created in the file `zzz.R` (this name ensures that it will always run after all
+the linters are defined). Simply add your linter name to the `default_linters`
 list before the `NULL` at the end.
```

Comment:
Probably should say that the default linters are meant for compliance to the tidyverse style, and that adding a default linter requires introducing a lint to default_linter_testcode.R.

Alternatively, we could remove this section since that's very {lintr} specific?

Reply:
I also considered whether we might want to drop it... I think on balance it's better to keep, for now. maybe eventually it belongs in a "CONTRIBUTING" document, though.

Reply:
Sounds good.